ovos_tts_plugin_server/__init__.py

```
@@ -42,48 +62,66 @@
+            data: bytes = self._fetch_audio_data_ovos(params, sentence, servers)
+        elif self.api == "openai":
+            params.pop("lang") # we can't set it in openai
+            if not model:
+                params["model"] = "default"
+            if not voice:
+                params["voice"] = "nova"
+            params["response_format"] = "wav"
+            if self.host:
+                servers = self.host
+            else: # we use official OpenAI service (paid!)
+                servers = "https://api.openai.com"
```

Ruff (0.11.9)

Use ternary operator `servers = self.host if self.host else "https://api.openai.com"` instead of `if`-`else`-block

(SIM108)